internal/controller/storagemgr/storage.go

```diff
@@ -74,8 +80,29 @@ func (s *StorageImpl) GetStatus() int {
 	return s.status
 }
 
+// checkMetadataStatus checks for metadata running to start DataStorage
+func checkMetadataStatus() bool {
+	ip, port, err := config.GetMetadataServerIP(dataStorageConfFolder + "/configuration.toml")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Is `return` unnecessary?

Reply:
return of GetMetadataServerIp or checkMetadataStatus?

Reply:
I meant there must be a 'return false' after the log.